### DISCUSSION TITLE
Removal of Jquery Migrate

### DISCUSSION THREAD

**Discussion by lingesh-k-7833** [jquery/jquery-migrate#510] [Q&A] [answered]
We are currently using jQuery 3.7.1 and jQuery Migrate version 3.4.0. Now we are planning to remove jQuery Migrate and proceed with jQuery alone. What are the steps to take and what should be noted before and after the removal of jQuery Migrate?

**Comment by mgol** [comment, marked as answer]
If you see no further warnings generated by Migrate, you should be able to just remove it. It's best to monitor the environment some time after the removal in case there's an edge case that wasn't covered by the warnings.
